src/twisted/internet/test/test_tcp.py

```diff
@@ -630,9 +632,23 @@ def makeConnection(self, transport):
         clientFactory.protocol = CheckAddress
 
         def connectMe():
-            reactor.connectTCP(
-                fakeDomain, server.getHost().port, clientFactory,
-                bindAddress=(self.interface, port))
+            while True:
```

Comment:
While unlikely, it's possible that this could run for a while.  Perhaps it would be good to retry only for a certain amount of time, after which the test would be `fail`ed?  This assumes you can trust `time.time`.

Reply:
If the host is so hosed that it's impossible to ever connectTCP without a ConnectBindError then I don't know what's going on.  I guess I'll leave this as-is.